### DISCUSSION TITLE
Why Yolo requires so many training epochs?

### DISCUSSION THREAD

**Discussion by Fan-Yixuan** [ultralytics/yolov5#7467] [Q&A] [answered]
Thank you for your excellent work! I would like to ask why Yolo generally needs to be trained for about 300 epochs, which is an order of magnitude larger than the 1x or 2x training schedule of the general two-stage detectors.

**Comment by zhiqwang** [comment, marked as answer]
YOLOv5 is trained from scratch, w/o the pretrained backbone from ImageNet datasets. And you can check torchvision's new training recipes as a comparison (They trained from scratch with 400 epochs, and get mAP 46.7 here).
- https://github.com/pytorch/vision/issues/5307
- https://github.com/pytorch/vision/pull/5715
- https://github.com/pytorch/vision/pull/5763#issue-1194515470

cc @glenn-jocher

**Reply by Fan-Yixuan** [reply]
Thanks for your quick reply and information!
